### DISCUSSION TITLE
Install nspanel pro tools

### DISCUSSION THREAD

**Discussion by dennisdenn070** [seaky/nspanel_pro_tools_apk#131] [Q&A]
hi i have a question, i installed my nspanel pro like the tutorial of mark watt. now i want my nspanel to work as a zigbee router. now i saw that you can do a lot with the nspanel pro tools. can i install the apk file of the nspanel pro tools without any problems?

**Comment by seaky** [comment]
With to original app and ewelink you can change the zigbee mode to router and you can pair your devices.
After that you can use it as a generic android device, zigbee mode will lasts even after reboot.

**Reply by dennisdenn070** [reply]
the existing firmware on my nspanel is 1.80, I do not have the function to switch to router, that only came later. Can I install your nspanel pro tools apk file on it and then still enable that router zigbee, I rooted it according to mark watt. thanks in advance

**Reply by seaky** [reply]
Sorry for late answer in this case you have to wait for next release which will enable it without fw upgrade!
Btw try to keep your device as lowest fw as possible because above 1.10 you can't use touch gestures above 3.0 you cant set proximity distance etc
